app/helpers/patternfly_components_helper.rb

```diff
@@ -23,21 +23,20 @@ def title_tag(title)
     end
   end
 
-  def body_tag(body)
-    tag.div class: 'pf-c-alert__description' do
-      tag.p body
-    end
-  end
+  def description_tag(description)
```

Comment:
I renamed `body` to `description` to make it more coherent, because in PF4 it's called "description".